UnleashedRecomp/main.cpp

```diff
@@ -211,6 +213,12 @@ int main(int argc, char *argv[])
         }
     }
 
+    if (!useDefaultWorkingDirectory)
+    {
+        // Set the current working directory to the executable's path.
+        std::filesystem::current_path(os::process::GetExecutablePath().parent_path());
```

Comment:
Should use the error code variant of this function, it'll throw an exception on failure otherwise.

Reply:
Done in c078d71.